src/main/java/core/basesyntax/interfaces/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface Drawable {
+    String draw();
```

Comment:
You should draw the Figure inside of method
```suggestion
    void draw();
```